frappe/utils/scheduler.py

```diff
@@ -25,15 +25,30 @@
 from frappe.core.doctype.user.user import STANDARD_USERS
 from frappe.installer import update_site_config
 from six import string_types
+from croniter import croniter
 
 DATETIME_FORMAT = '%Y-%m-%d %H:%M:%S'
 
+SOBSTITUTIONS = {
+    "yearly": "0 0 1 1 *",
+    "anually": "0 0 1 1 *",
+    "monthly": "0 0 1 * *",
+    "monthly_long": "0 0 1 * *",
+    "weekly": "0 0 * * 0",
+    "weekly_long": "0 0 * * 0",
+    "daily": "0 0 * * *",
+    "daily_long": "0 0 * * *",
+    "midnight": "0 0 * * *",
+    "hourly": "0 * * * *",
+    "hourly_long": "0 * * * *",
+    "all": "0/" + str((frappe.get_conf().scheduler_interval or 240) // 60) + " * * * *",
```

Comment:
what is `// 60` ?

Reply:
`// 60`  is a integer division to get minutes.